### DISCUSSION TITLE
Vercel API Endpoint for Notifications

### DISCUSSION THREAD

**Discussion by versecafe** [vercel/community#6433] [Help]
### Summary

I can't find the endpoints for grabbing the data contained in the notifications menu in Vercel from the REST API, I'm not sure if I'm just missing the data, if the endpoint isn't documented, or if one currently doesn't exist.

@amyegan would you have any idea on this?

### Example

https://vercel.com/docs/rest-api/endpoints

### Steps to Reproduce

Lack of information no steps to reproduce

**Comment by versecafe** [comment]
@mcsdevv Would you have any idea on this, I''m trying to get the data for [Tiny Triangle](github.com/versecafe/TinyTriangle) to add support for seeing new notifications to it, I'm not sure if the API was ever intended to support this or if it was just made for more integration like services such as Planet Scales integration.
 
![image](https://github.com/vercel/community/assets/147033096/ace84f56-fb19-49a6-8c86-13c2635b69ee)

**Comment by amyegan** [comment]
Hey @versecafe. It doesn't look like we have a public API for notifications right now. But I'm checking with the notifications team to see if there are any undocumented options that might be accessible. If you send an email to amy.egan@vercel.com, I can follow up with you there when I have more info 🙏

**Reply by versecafe** [reply]
Sent an email, thank you
